common/src/main/java/org/apache/celeborn/common/write/SlowStartPushStrategy.java

```diff
@@ -108,7 +108,7 @@ public int getContinueCongestedNumber() {
 
   public SlowStartPushStrategy(CelebornConf conf) {
     super(conf);
-    this.maxInFlight = conf.clientPushMaxReqsInFlight();
+    this.maxInFlight = conf.clientPushMaxReqsInFlightPerWorker();
```

Comment:
should we update the related variable and method name to make it clearer?

Reply:
done